docs/crocks/Reader.md

```diff
@@ -0,0 +1,61 @@
+# Reader
+
+`Reader a`
+
+--
+
+--
+
+```js
+--
+```
+
+`Reader` exposes the following functions on the constructor and instance:
+
+| Constructor | Instance |
+|:---|:---|
+| [`ask`](#ask), [`of`](#of), [`type`](#type) |  [`ap`](#ap), [`chain`](#chain), [`inspect`](#inspect), [`map`](#map), [`of`](#of), [`run`](#runWith), [`type`](#type) |
+
+## Constructor
+
+### ask
+
+`Reader m => a -> m a`
+
+### of
+
+`Reader m => a -> m a`
+
+### type
+
+`() -> String`
+
+## Instance
+
+### ap
+
+`Reader m => m (a -> b) ~> m a -> m b`
+
+### chain
+
+`Reader m => m a ~> m a -> (a -> m b) -> m b`
+
+### inspect
+
+`() -> String`
+
+### map
+
+`Reader m => m a ~> (a -> b) -> m b`
+
+### of
+
+`Reader m => a -> m a`
```

Comment:
If something applies to all the sigs, if you prefer you can rather add it to the review note instead of to each line